From: Matthias Clasen <mclasen@redhat.com>
Date: Sat, 1 Aug 2020 22:42:25 -0400
Subject: cups: Don't emit a warning if we can't talk to avahi

The g_warning in this case makes the templates test
fail if it can't connect to the system bus (as is the
case for running installed tests in ci).

Origin: upstream, 3.99.1
---
 modules/printbackends/gtkprintbackendcups.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/printbackends/gtkprintbackendcups.c b/modules/printbackends/gtkprintbackendcups.c
index 6453e82..5ca8049 100644
--- a/modules/printbackends/gtkprintbackendcups.c
+++ b/modules/printbackends/gtkprintbackendcups.c
@@ -3413,7 +3413,7 @@ avahi_create_browsers (GObject      *source_object,
   if (!dbus_connection)
     {
       if (!g_error_matches (error, G_IO_ERROR, G_IO_ERROR_CANCELLED))
-        g_warning ("Couldn't connect to D-Bus system bus, %s", error->message);
+        g_message ("Couldn't connect to D-Bus system bus, avahi printers will not be available: %s", error->message);
 
       g_error_free (error);
       return;
